template/README.md

```diff
@@ -11,27 +11,97 @@ If you found this in `fredrikhgrelland/vagrant-hashistack`, you may be intereste
 
 Documentation on [parent repository](https://github.com/fredrikhgrelland/vagrant-hashistack#usage).
 
-## Customizing and using the vagrant box
+## Goals of this template
+This template aims to supply a developer with the tools necessary to do three things:
+- Build a docker image
+- Build a terraform module
+- Test and develop these two within a hashistack ecosystem
+
+To achieve this we utilise a vagrant-box built from [fredrikhgrelland/vagrant-hashistack](https://github.com/fredrikhgrelland/vagrant-hashistack/).
+
+## Where do I start?
+First of all I would recommend you read the section ["Using the template and box"](using-the-template-and-box) for a detailed look at the box's features and configurations. Below is a rough step-by-step going from a fresh template to a functioning terraform module. You can find example-code of everything below in [test_example/](test_example/).
+
+Step-by-step:
+1. Create a [Dockerfile](https://docs.docker.com/engine/reference/builder/) and put it in `docker/`.
+1. Use the box's startup playbooks to build and make the image available within the box. [Startup scheme](default-startup-scheme). [Example building image](test_example/dev/ansible/01_build_docker_image.yml).
+1. Create a [Nomad job specification](https://www.nomadproject.io/docs/job-specification). This should be placed in [conf/nomad/](conf/nomad/README.md). [Example hcl-file](test_example/conf/nomad/countdash.hcl)
+1. Use terraform to deploy this job to Nomad. Refer to ["Terraform module requirements"](terraform-module-requirements) on how to do this.
```

Comment:
Is it `(terraform-module-requirements)` mock right now?

Reply:
Are you asking if the `Terraform module requirements` section is a mock?

Reply:
Question is related to other question above

Reply:
Right, as above: written wrong, been fixed